hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -218,23 +226,27 @@ struct PixelShader {
 static void add_var_ref(struct PixelShader *ps, const char *var)
 {
     int i;
-    for (i=0; i<ps->num_var_refs; i++) {
-        if (strcmp((char*)ps->var_refs[i], var) == 0) return;
+    for (i = 0; i < ps->num_var_refs; i++) {
+        if (strcmp((char *)ps->var_refs[i], var) == 0) {
+            return;
+        }
     }
-    strcpy((char*)ps->var_refs[ps->num_var_refs++], var);
+    strcpy((char *)ps->var_refs[ps->num_var_refs++], var);
```

Comment:
I'm a bit surprised, what type is `var_refs`? Why do we even need a cast? This looks sketchy.

*Not your change, but still had me wondering.*